packages/ketcher-core/src/domain/entities/DrawingEntitiesManager.ts

```diff
@@ -116,6 +117,37 @@ export class DrawingEntitiesManager {
     return command;
   }
 
+  public moveSelectedDrawingEntities(offset: Vec2) {
+    const command = new Command();
+
+    this.monomers.forEach((drawingEntity) => {
+      if (drawingEntity.selected) {
+        command.merge(
+          this.createDrawingEntityMovingCommand(drawingEntity, offset),
+        );
+      }
+    });
+
+    this.polymerBonds.forEach((drawingEntity) => {
+      command.merge(
+        this.createDrawingEntityMovingCommand(drawingEntity, offset),
+      );
+    });
```

Comment:
Please mode only selected bonds and those which are connected to selected monomers

Reply:
Now it it is better, thank you!